internal/server/data/import.go

```diff
@@ -32,36 +30,30 @@ func Import(
 	in *pb.ImportRequest,
 	st *store.Store,
 	metadata *resource.Metadata,
-	openSql bool,
 ) (*resource.Cache, error) {
-	var err error
-	if err = writer.Write(
+	// First clear the tables.
+	err := sqldb.ClearTables(st.SQLClient, metadata.SQLitePath)
```

Comment:
Is it worthwhile backing up vs deleting tables?

Reply:
I think eventually yes. I can imagine there is an endpoint for user to backup a table and have it in a "admin" UI. This is more of a prototype workflow for now.